packages/java/endpoint/src/main/java/com/vaadin/hilla/signals/Signal.java

```diff
@@ -219,4 +224,8 @@ public int hashCode() {
     public static void setMapper(ObjectMapper mapper) {
         StateEvent.setMapper(mapper);
     }
+
+    private Logger getLogger() {
```

Comment:
Does it have any difference with the original `static` property implementation? 🤔

Reply:
It makes testing easier, while the static thing was very hard to spy on.